packages/phone-number-privacy/combiner/src/domain/endpoints/sign/action.ts

```diff
@@ -1,19 +1,14 @@
 import {
-  CombinerEndpoint,
   DomainRestrictedSignatureRequest,
   ErrorMessage,
   ErrorType,
-  getSignerEndpoint,
-  SignerEndpoint,
   WarningMessage,
 } from '@celo/phone-number-privacy-common'
 import { CryptoSession } from '../../../common/crypto-session'
 import { SignAction } from '../../../common/sign'
 import { DomainDiscrepanciesLogger } from '../../services/log-discrepancies'
 
 export class DomainSignAction extends SignAction<DomainRestrictedSignatureRequest> {
-  readonly endpoint: CombinerEndpoint = CombinerEndpoint.DOMAIN_SIGN
```

Comment:
it's both unused & IMO it's an antipattern to have it on this specific Action class since it's already part of `io`, so it can already be accessed via `this.io.endpoint`. (It's also not part of any other Actions and only part of IOs).